### PR TITLE
Recommendations for user and ability to create playlist

### PR DESCRIPTION
**Pull request by vandyliu**
idk why it took so long to load so sorry for long gif

### CODE REVIEW

File: backend/controller/SpotifyController.js [hunk 1]
```diff
@@ -123,6 +123,98 @@ exports.getMyRecentlyPlayedTracks = (spotifyId, options) => {
...
+  }).then(
+    (data) => { return data; },
+    (_err) => {
+      console.log('Something went wrong!', err);
+      throw err;
+    });
+}
+
+/**
+ * Gets recomendations based on most recently played tracks
```
Comment (ianmah) [inline backend/controller/SpotifyController.js:175]: yoink

**Review by ianmah — COMMENTED** [review]
I think it wouldve been nice to have the spotifycontroller / create a client wrapper to automatically handle the refresh token (since we have to do that logic every time it seems) but thats a problem for another day.

LGTM 🚀

**Review by ianmah — APPROVED** [review]

File: backend/controller/SpotifyController.js (hunk 1 above)
Comment (vandyliu) [reply backend/controller/SpotifyController.js]: I got a surprise for you 🤭

Test it though, It works on my end but I'm never done code like this LOL

**Review by ianmah — APPROVED** [review]

**Comment by ianmah** [comment]
pog

**Pull request merged**